com.sublimetext.three.appdata.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018 Endless Mobile, Inc. -->
+<component type="desktop">
+  <id>com.sublimetext.three</id>
```

Comment:
This might be bikeshedding but i do find that an odd id. Maybe `com.sublimetext.SublimeText3` makes more sense?

Reply:
I'll admit that I kind of love the naming. But I'm probably not to be trusted with a love of whimsy.

Reply:
I don't much like the ID myself, but it's the app ID from upstream, and the program tries to own that bus name, so I think it should stay.